src/app/utils/bookStorage.ts

```diff
@@ -47,7 +47,8 @@ export function saveBook(book: Book) {
 }
 
 export function getBooks() {
-    return JSON.parse(localStorage.getItem(localStorageItem));
+    const books = JSON.parse(localStorage.getItem(localStorageItem))
+    if (localStorageItem) return books;
```

Comment:
Did you mean to use `books` here? `localStorageItem` will always be true because it's a constant